### DISCUSSION TITLE
Increase the presence of "q" , "z" and "x" letters in the basic "english" mode

### DISCUSSION THREAD

**Discussion by dzintars** [monkeytypegame/monkeytype#1045] [Ideas]
IMO simple "english" mode is where many new users land at the very beginning.
The issue is that in that mode letters "q" , "z" and "x" are not included https://github.com/Miodec/monkeytype/blob/master/static/languages/english.json
I think, this should be fixed.
The words which come into the mind could be:
```json
{
  "words":  [
    "query",
    "zero",
    "quantity",
    "qualify",
    "next",
    "example",
    "wax",
    "muzzle",
    "maze",
    "freeze",
    "unique",
    "squirrel",
  ]
}
```

**Comment by Miodec** [comment]
Yeah people suggested that, the issue is that adding more words to the word list right now, would effectively make it harder, which makes all the results on the leaderboard kinda invalid since they were achieved on the easier list.

Im not sure what is a good solution here.

**Reply by frogfrogfrogfrogfrog** [reply]
Wait isn't "simple english" simply the top 100 words? So the top 100 words just don't have q x and z right? Therefore if you add words then they'd have to be past the top 100 words which doesn't make sense. I think a solution could be just having separate leaderboards for the "simple english" and other word sets like English 1k. Or whenever someone completes a score on the leaderboard it could say which word set they used. So an example would be:

Name, wpm, raw, test, date, language

That way it includes other english sets such as 450k while also including simple english. It could also technically include other languages.

edit: it's the top 200

**Reply by Miodec** [reply]
English is 200 words.

**Reply by frogfrogfrogfrogfrog** [reply]
:/

well i mean the idea is still there
if you increase the word set it'll be past the 200 words

**Reply by Miodec** [reply]
As i said though, im more worried about making it harder for everyone by adding more words compared to people who already got good scores on the leaderboard than having a few more words over the 200 mark.

**Reply by dzintars** [reply]
But how fair are those scores if they are missing ~10% of the letters? IMHO few words should be swapped keeping the character count.

**Reply by neinja007** [reply]
What about making these letters more common only for users t´who activate a specific setting?

this could be called, something like "key frequency" and it is either "balanced" (each key has roughly the same probability of coming up) or "real" (the top 250 english words are used, period. no letter adjustment.)

these names are pretty terrible, but i hope you get my point.

**Reply by fehmer** [reply]
with `q` having 0.095%, `x` having 0.15% and `z` having 0.074% relative frequency we are not missing 10% of the letters.
Source: https://en.wikipedia.org/wiki/Letter_frequency

**Comment by dzintars** [comment]
You guys definitely know the background better, so i can speak just from my "simple user" perspective.
With "simple English" mode personally i do not care much about the leader-boards as i don't think it's carries any meaningful value besides just a simple fun. Where the leader-boards really make sense is the "1K+" modes.
I treat "simple English" (100 words) just as training space for the beginners (like me) and i am interested to gain the real typing skill instead of being in some of the leader-board. What's the point of leader-board for 100 common words?
This definitely is just my SUBJECTIVE OPINION.
Mby it is reasonable to consider swapping last few words for the next common words which does include those letters? So that those letters appear in the test at least once or twice. In my opinion all letters should be included in any given test. Leader-board feature should be less important than "teach to type" feature. Learning should be the priority and the competition goes after.

**Reply by frogfrogfrogfrogfrog** [reply]
The problem with this is that Monkeytype is designed (in my subjective opinion) in a more competitive environment where you can hone your typing skills. The leaderboard can be for fun but for some people it's a genuinely serious object to work towards. I don't know about you or other people but to solve this problem you could simply... Switch to 1k+ modes? Since people complain about there not being q, z, or x then they won't complain about the harder words. Besides, you'll type the harder words faster as well.

**Reply by dzintars** [reply]
This is what i did (switched to 1K) - after i realized, that there is just a small set of words by default.
